massive-data-import/README.md

```diff
@@ -0,0 +1,152 @@
+  .then(() => {
+    console.log(`${bulkQuery.body.bulkData.length / 2} lines inserted`);
+  })
+```
+
+The second method is about half as fast but it allows you to benefit from all the usual features of Kuzzle.  
+
+It uses the [Document Controller's mCreate](https://docs.kuzzle.io/api-documentation/controller-document/m-create/) action to insert multiple documents into the same query.  
+The creation of documents will send notifications to customers who have subscribed to a request corresponding to them.  
+
+The documents to be inserted will have to be collected in a table before being passed to the [SDK method](https://docs.kuzzle.io/sdk-reference/collection/mcreate-document/) corresponding to the mCreate action of the Document Controller :
+
+```js
+const documents = [
+  {
+    pickup_datetime:  '03/01/2016 12:10:29 AM',
+    dropoff_datetime: '03/01/2016 12:19:05 AM',
+    passenger_count:  '5',
+    trip_distance:    '2.51',
+    pickup_position:  { lon: '-73.993759155273438', lat: '40.718101501464844' },
+    dropoff_position: { lon: '-73.951248168945312', lat: '40.711910247802734' },
+    fare_amount:      '11.3'
+  },
+  {
+    pickup_datetime:  '03/01/2016 12:10:28 AM',
+    dropoff_datetime: '03/01/2016 12:21:56 AM',
+    passenger_count:  '1',
+    trip_distance:    '2.5',
+    pickup_position:  { lon: '-73.974746704101563', lat: '40.793346405029297' },
+    dropoff_position: { lon: '-73.979293823242187', lat: '40.761508941650391' },
+    fare_amount:      '11.8'
+  }
+]
+
+kuzzle
+  .collection('yellow-taxi', 'nyc-open-data')
+  .mCreateDocumentPromise(documents)
+  .then(() => {
+    console.log(`${documents.length} lines inserted`);
+  })
+  .catch(error => {
+    console.dir(error, {depth: null, colors: true});
+    process.exit(1);
+  });
+```
+
+## Try it yourself
+
+You can use the [docker-compose.yml](docker-compose.yml) included in this How-To to test the massive data import.  
+The containers are preconfigured to work with NYC Open Data's Yellow Taxi dataset.
+
+```bash
+docker-compose up
+```
+
+Then in another terminal we are going to subscribe to a room with [geoBoundingBox](https://docs.kuzzle.io/kuzzle-dsl/terms/geo-bounding-box/) corresponding to the Time Square area. We will receive a notification each time a new document corresponds to a passenger dropped in this area.  
+
+```bash
+docker-compose exec kuzzle node /scripts/subscribe.js
+```
+
+In another terminal we are going to import the dataset with the two methods.
+
+```bash
+time docker-compose exec kuzzle node /scripts/load_bulk.js
+
+time docker-compose exec kuzzle node /scripts/load_mcreate.js
+```
+
+On a laptop with a I5-7300U CPU @ 2.60 GHz, 16GiB of RAM and a SSD it takes approximatively 1 minutes to load 1 millions of document in Kuzzle with the Bulk Api method and approximatively 2 minutes with mCreate method.  
+
+As we can see, although the bulk method is twice as fast as the mcreate method, it doesn't trigger any subscription notification.  
```

Comment:
remove "As we can see,"

just:
Although the bulk method is twice as fast as the mCreate method, it doesn't trigger any subscription notification.